processing/src/main/java/io/druid/segment/GenericColumnSerializer.java

```diff
@@ -20,20 +20,14 @@
 package io.druid.segment;
 
 import io.druid.guice.annotations.ExtensionPoint;
-import io.druid.java.util.common.io.smoosh.FileSmoosher;
+import io.druid.segment.serde.Serializer;
 
-import java.io.Closeable;
 import java.io.IOException;
-import java.nio.channels.WritableByteChannel;
 
 @ExtensionPoint
-public interface GenericColumnSerializer extends Closeable
+public interface GenericColumnSerializer extends Serializer
 {
   public void open() throws IOException;
```

Comment:
Please remove the public modifier.

Reply:
@jihoonson could you please add an IntelliJ/Checkstyle/PMD rule that prohibits unnecessary qualifiers in interfaces?

Reply:
Ok. I'll make a PR soon.